jumpscale/servers/threebot/threebot.py

```diff
@@ -416,14 +427,15 @@ def add(self, path: str = None, giturl: str = None, **kwargs):
     def delete(self, package_name):
         if package_name in DEFAULT_PACKAGES:
             raise j.exceptions.Value("cannot delete default packages")
-
         package = self.get(package_name)
         if not package:
             raise j.exceptions.NotFound(f"{package_name} package not found")
 
         # remove bottle servers
-        for bottle_server in package.bottle_servers:
-            self.threebot.rack.remove(f"{package.name}_{bottle_server['name']}")
+        for bottle_server in self.threebot.rack._servers:
+            if package_name in bottle_server:
```

Comment:
this check is not enough 
for example you have two packages *farm* & *farm_manager* and you want to remove package *farm*
with this condition the servers of package *farm_manager* will be deleted to because farm in farm_manager
you need to do something like
```python
if bottle_server.startswith(f"{package_name}_"):
```